perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1384,19 +1384,28 @@ sub user_add : Path('user_edit') : Args(0) {
     my $email_v = $c->get_param('email_verified');
     my $phone_v = $c->get_param('phone_verified');
 
-    unless ($email || $phone) {
-        $c->stash->{field_errors}->{username} = _('Please enter a valid email or phone number');
-    }
-    if (!$email_v && !$phone_v) {
-        $c->stash->{field_errors}->{username} = _('Please verify at least one of email/phone');
-    }
     if ($email && !is_valid_email($email)) {
         $c->stash->{field_errors}->{email} = _('Please enter a valid email');
     }
     unless ($c->get_param('name')) {
         $c->stash->{field_errors}->{name} = _('Please enter a name');
     }
 
+    if ( $c->get_param('send_login_email') ) {
```

Comment:
So this is for if they fill in a bit of the Add User form in the admin, but then use it to send a login email, same as if they filled in `/auth` on behalf of someone else, I guess? And the rest of the form (apart from name) is ignored. Is it worth having this, or is it clearer if it's only on the user edit page, do you think?

Reply:
I just thought it was a bit more obvious to use this form if they want to add a user rather than add user and then send them a log in email. Although I can see arguments the other way too. I don't really have strong feelings about it.